src/test/suite/followLink.test.ts

```diff
@@ -0,0 +1,73 @@
+import * as assert from "assert";
+import * as vscode from "vscode";
+import * as sinon from "sinon";
+import * as os from "os";
+import { openNewEditor } from "../openNewEditor";
+import { getFixturePath } from "./getFixturePaths";
+
+suite("followLink", async function () {
+  this.timeout("100s");
+  this.retries(3);
+
+  teardown(() => {
+    sinon.restore();
+  });
+
+  test("follow definition", followDefinition);
+  test("follow link", followLink);
+});
+
+async function followDefinition() {
+  const editor = await openNewEditor(
+    "const foo = 'hello';\nconst bar = foo;",
+    "typescript"
+  );
+  await vscode.commands.executeCommand("revealLine", {
+    lineNumber: 1,
+    at: "top",
+  });
+  editor.selections = [new vscode.Selection(1, 12, 1, 15)];
+
+  assert.equal(editor.visibleRanges[0].start.line, 1);
+
+  await vscode.commands.executeCommand(
+    "cursorless.command",
+    "follow this",
+    "followLink",
+    [
+      {
+        type: "primitive",
+        mark: {
+          type: "cursor",
+        },
+      },
+    ]
+  );
+
+  assert.equal(editor.visibleRanges[0].start.line, 0);
+}
+
+async function followLink() {
+  const filename = getFixturePath("helloWorld.txt");
+  const link =
+    os.platform() === "win32" ? `file:///${filename}` : `file://${filename}`;
+  await openNewEditor(link);
```

Comment:
might be worth a quick comment here and/or renaming the link variable, otherwise it looks a like you're opening the link, rather than creating a file whose text is the link